models/token.js

```diff
@@ -17,7 +17,7 @@ const MODEL = {
 
     hash: Joi
         .string()
-        .base64()
+        .regex(/[a-zA-Z0-9_-]+~/)
```

Comment:
[Looking at `base64url`](https://github.com/brianloveswords/base64url/blob/master/src/base64url.ts#L27-L29), it translates the `=` into nothing. So your character-set choice seems valid. 

I'd recommend a comment here that it's based on this library's translation table, in case someone in the future decides to use a different library.